src/assets/i18n/pl.po

```diff
@@ -575,70 +575,70 @@ msgid ""
 msgstr ""
 
 msgid "ACL Mode"
-msgstr "Tryb ACL"
+msgstr ""
 
 msgid "ACL Type"
-msgstr "Typ ACL"
+msgstr ""
 
 msgid "ACL cannot be changed when the Source is the top-level dataset for a pool."
 msgstr ""
 
 msgid "ACME DNS"
-msgstr "ACME DNS"
+msgstr ""
 
 msgid "ACME Server Directory URI"
-msgstr "ACME Server Directory URI"
+msgstr ""
 
 msgid "AD"
-msgstr "AD"
+msgstr ""
 
 msgid "AD Timeout"
-msgstr "AD przekroczono maksymalny dozwolony czas"
+msgstr ""
 
 msgid "AFP"
-msgstr "AFP"
+msgstr ""
 
 msgid "AFP3 Unix Privs"
-msgstr "AFP3 Unix Privs"
+msgstr ""
 
 msgid "API"
-msgstr "API"
+msgstr ""
 
 msgid "API Key"
-msgstr "Klucz API"
+msgstr ""
 
 msgid "API Key or Password"
-msgstr "Klucz API lub Hasło"
+msgstr ""
 
 msgid "API URL"
-msgstr "API URL"
+msgstr ""
 
 msgid "APPLY CHANGES"
-msgstr "ZATWIERDŹ ZMIANY"
+msgstr ""
 
 msgid "ARN"
-msgstr "ARN"
+msgstr ""
 
 msgid "ATA Security User"
-msgstr "Użytownik bezpieczeństwa ATA"
+msgstr ""
 
 msgid "AUTORID"
-msgstr "AUTORID"
+msgstr ""
 
 msgid "AWS Region"
-msgstr "Region AWS"
+msgstr ""
 
 msgid "About"
 msgstr ""
 
 msgid "Access"
-msgstr "Dostęp"
```

Comment:
I notice this PR removes some translations, was that intentional?

Reply:
From what I see everything is as it should be. Unless I look wrong? hmm

Reply:
if you look at what it adds in in its place all you see is a "", I think your PR is somehow removing these, perhaps you were working off an older version of this file?